ingestion/src/metadata/test_suite/api/workflow.py

```diff
@@ -155,11 +155,23 @@ def _get_service_connection_from_test_case(self, table_fqn: str):
                     )
                 )
                 service_connection_config = deepcopy(service_connection.__root__.config)
-                if (
-                    hasattr(service_connection_config, "supportsDatabase")
-                    and not service_connection_config.database
-                ):
-                    service_connection_config.database = table_fqn.split(".")[1]
+                if hasattr(service_connection_config, "supportsDatabase"):
```

Comment:
would it make sense to have this moved out to a utils or something? Feels like a generic "get the db from a table depending on the source" kind of thing

Reply:
Yes definitely. I am thinking of actually combining part of the profiler and test suite workflow as a lot of pieces are similar. I think we should do it separately from this though -- thinking to do the lest amount of changes for the minor release.

Reply:
oh that's just for a fix pls go for it

Reply:
thanks mate